deep_qa/layers/backend/envelope.py

```diff
@@ -49,7 +49,7 @@ def compute_output_shape(self, input_shape):
     @overrides
     def call(self, inputs, mask=None):
         span_begin, span_end = inputs
-        after_span_begin = cumulative_sum(span_begin)
-        after_span_end = cumulative_sum(span_end)
+        after_span_begin = K.cumsum(span_begin)
```

Comment:
@matt-gardner  The error I'm getting in Travis: `Module 'keras.backend' has no 'cumsum' member (no-member)`